opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/WildcardQueryTest.java

```diff
@@ -0,0 +1,102 @@
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.opensearch.sql.common.antlr.SyntaxCheckException;
+import org.opensearch.sql.data.model.ExprValue;
+import org.opensearch.sql.data.type.ExprType;
+import org.opensearch.sql.exception.SemanticCheckException;
+import org.opensearch.sql.expression.DSL;
+import org.opensearch.sql.expression.Expression;
+import org.opensearch.sql.expression.FunctionExpression;
+import org.opensearch.sql.expression.NamedArgumentExpression;
+import org.opensearch.sql.expression.config.ExpressionConfig;
+import org.opensearch.sql.expression.env.Environment;
+import org.opensearch.sql.expression.function.FunctionName;
+import org.opensearch.sql.opensearch.storage.script.filter.lucene.relevance.WildcardQuery;
+
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+class WildcardQueryTest {
+  private static final DSL dsl = new ExpressionConfig()
+      .dsl(new ExpressionConfig().functionRepository());
+  private final WildcardQuery wildcardQueryQuery = new WildcardQuery();
+  private final FunctionName wildcardQueryFunc = FunctionName.of("wildcard_query");
+
+  static Stream<List<Expression>> generateValidData() {
+    return Stream.of(
+        List.of(
+            dsl.namedArgument("field", DSL.literal("title")),
+            dsl.namedArgument("query", DSL.literal("query_value*")),
+            dsl.namedArgument("boost", DSL.literal("0.7")),
+            dsl.namedArgument("case_insensitive", DSL.literal("false")),
+            dsl.namedArgument("rewrite", DSL.literal("constant_score_boolean"))
+        )
+    );
+  }
+
+  @ParameterizedTest
+  @MethodSource("generateValidData")
+  public void test_valid_parameters(List<Expression> validArgs) {
+    Assertions.assertNotNull(wildcardQueryQuery.build(
+        new WildcardQueryExpression(validArgs)));
+  }
+
+  @Test
+  public void test_SyntaxCheckException_when_no_arguments() {
+    List<Expression> arguments = List.of();
+    assertThrows(SyntaxCheckException.class,
+        () -> wildcardQueryQuery.build(new WildcardQueryExpression(arguments)));
+  }
+
+  @Test
+  public void test_SyntaxCheckException_when_one_argument() {
+    List<Expression> arguments = List.of(namedArgument("field", "title"));
+    assertThrows(SyntaxCheckException.class,
+        () -> wildcardQueryQuery.build(new WildcardQueryExpression(arguments)));
+  }
+
+  @Test
+  public void test_SemanticCheckException_when_invalid_parameter() {
+    List<Expression> arguments = List.of(
+        namedArgument("field", "title"),
+        namedArgument("query", "query_value*"),
+        namedArgument("unsupported", "unsupported_value"));
+    Assertions.assertThrows(SemanticCheckException.class,
+        () -> wildcardQueryQuery.build(new WildcardQueryExpression(arguments)));
+  }
+
+  private NamedArgumentExpression namedArgument(String name, String value) {
```

Comment:
Similar to the comment above -- you can just `import static ... DSL.namedArgument`  and remove this helper.

Reply:
Fixed in 0b9752ca994a334de8e9bbc6bd7ac49fadd37944